### DISCUSSION TITLE
Compatibility with Squarespace

### DISCUSSION THREAD

**Discussion by JakobGlavak** [umputun/remark42#923] [Q&A]
A quick question: how do embed Remark42's comment section into my Squarespace blog? Squarespace makes website building easy for us who know nothing about coding, but it also makes code insertion difficult. Everything comes with a price, I suppose.

Keep in mind, I don't know anything about coding, so explain it to me like I'm five. :)

**Comment by umputun** [comment]
> how do embed Remark42's comment section into my Squarespace blog?

No clue, never used it. To me, it looks like a hosting service allowing you to build and host sites with them. As long as you have remark42 running somewhere (i.e. Digital Ocean, Linode, etc) and able to inject custom JS into generated pages you should be able to do it.

> Keep in mind, I don't know anything about coding, so explain it to me like I'm five. :)

Pls see README providing step-by-step instruction on how to install remark42

**Reply by paskal** [reply]
If somebody does it, please contribute the doc for installation so less experienced users will benefit.
